snap/info.go

```diff
@@ -91,11 +92,11 @@ type PlaceInfo interface {
 
 	// DataHomeDir returns a glob that matches all per user data directories
 	// of a snap.
-	DataHomeDir(opts *dirs.SnapDirOptions) string
+	DataHomeDir(opts *dirs.SnapDirOptions) []string
```

Comment:
Maybe it needs to be renamed to `DataHomeDirs` with the comment updated to describe the new behavior.